bigframes/remote_function.py

```diff
@@ -516,17 +516,14 @@ def remote_function_node(
     """Creates an Ibis node representing a remote function call."""
 
     fields = {
-        name: rlz.value(type_) if type_ else rlz.any
+        name: rlz.ValueOf(None if type_ == "ANY TYPE" else type_)
```

Comment:
Actually we are using almost the same code internally, possibly it could worth to expose it in a more developer friendly way after we understand the actual requirements. We can open an issue on our side to track this.

Reply:
True. I was hoping to upstream this sooner, but now that I see https://ibis-project.org/reference/scalar-udfs#ibis.expr.operations.udf.scalar.builtin maybe I can migrate to that? I'll try that in a separate PR.

Reply:
Started on this here: https://github.com/googleapis/python-bigquery-dataframes/pull/277

I found a few issues already with UDF in the BQ backend. I'll get those documented soon.